mentat/command/commands/load.py

```diff
@@ -0,0 +1,63 @@
+from pathlib import Path
+from typing import List
+import json
+
+from typing_extensions import override
+
+from mentat.auto_completer import get_command_filename_completions
+from mentat.command.command import Command, CommandArgument
+from mentat.session_context import SESSION_CONTEXT
+from mentat.utils import mentat_dir_path
+from mentat.errors import PathValidationError
+
+
+class LoadCommand(Command, command_name="load"):
+    @override
+    async def apply(self, *args: str) -> None:
+        session_context = SESSION_CONTEXT.get()
+        stream = session_context.stream
+        code_context = session_context.code_context
+        context_file_path = mentat_dir_path / "context.json"
+
+        if len(args) > 1:
+            stream.send(
+                "Only one context file can be loaded at a time", style="warning"
+            )
+            return
+
+        if not args:
+            stream.send(
+                "No context file specified. Defaulting to context.json", style="warning"
+            )
+        else:
+            try:
+                context_file_path = Path(args[0]).expanduser().resolve()
+            except RuntimeError as e:
+                raise PathValidationError(
+                    f"Invalid context file path provided: {args[0]}: {e}"
+                )
+
+        with open(context_file_path, "r") as file:
+            parsed_include_files = json.load(file)
+
+        # TODO: Do we remove already-included files when loading new context file?
```

Comment:
I don't think this is necessary. The user can always run `/exclude *` first or restart mentat if that's what they want. But if we make the default the other way there won't be any easy way to add the context from the file to context.